docs/src/main/sphinx/connector/jdbc-procedures.fragment

```diff
@@ -0,0 +1,12 @@
+Procedures
+^^^^^^^^^^
+
+* ``system.flush_metadata_cache()``
+
+  Flush JDBC Metadata caches. For example below query will flush
```

Comment:
metadata caches. For example, the following system call flushes the metadata caches for the myschema schema in the example catalog

Reply:
Also ... is that true or does it flush the caches for all schemas?

Reply:
Yes procedure is registered in context of single catalog @findepi can you confirm to have 100%

Reply:
yes, all schemas within a catalog